runelite-client/src/main/java/net/runelite/client/plugins/blastfurnace/BlastFurnacePlugin.java

```diff
@@ -50,12 +59,17 @@
 {
 	private static final int BAR_DISPENSER = NULL_9092;
 
+	private static final Pattern FOREMAN_PERMISSION_PATTERN = Pattern.compile("Okay, you can use the furnace for ten minutes. Remember, you only need half as much coal as with a regular furnace.");
```

Comment:
Why are you using a `Pattern` for a static string?

Reply:
Resolved in d642d6b